modules/fun/fun.h

```diff
@@ -103,7 +103,7 @@ class CPlayer
 
 	private:
 
-		int  bodyHits_[kMaxClients + 1] {};
+		int  bodyHits_[kMaxClients + 1];
```

Comment:
How is it related to this PR?

Reply:
Not sure. It's possible I was playing with PVS-Studio and forgot about this change.